src/CoreBundle/private/library/classes/dbobjects/TCMSContentBoxItem.class.php

```diff
@@ -78,16 +80,16 @@ public function loadMenuItems()
 
             $oCMSModuleList = TdbCmsModuleList::GetList($query, $this->iLanguageId);
             while ($oCMSModule = $oCMSModuleList->Next()) {
-                $tableInUserGroup = $activeUser->oAccessManager->user->IsInGroups($oCMSModule->fieldCmsUsergroupId);
-                if ($tableInUserGroup) {
-                    $oMenuItem = new TCMSMenuItem_Module();
-                    $oMenuItem->SetData($oCMSModule->sqlData);
+                if (false === $securityHelper->isGranted(CmsPermissionAttributeConstants::ACCESS, $oCMSModule)) {
```

Comment:
Can this method handle a CMSModule as input?

Reply:
yes - via `\ChameleonSystem\SecurityBundle\Bridge\Chameleon\TableObject\CmsModule`